lib/trufi_models.dart

```diff
@@ -366,6 +366,7 @@ class PlanItineraryLeg {
     this.distance,
     this.duration,
     this.toName,
+    this.agencyName
```

Comment:
```suggestion
    this.agencyName,
```
We leave an extra `,` at the end for automatic code formatting.